torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5018,6 +5024,19 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            supports_gradgrad=False,
            supports_out=False,
            autodiff_nonfusible_nodes=["aten::hardswish"]),
+    OpInfo('nn.functional.leaky_relu',
+           aliases=None,
+           aten_name="leaky_relu",
+           dtypes=floating_types(),
+           sample_inputs_func=sample_inputs_leaky_relu,
+           dtypesIfCUDA=floating_types_and(torch.float16, torch.bfloat16),
+           supports_autograd=True,
+           assert_autodiffed=True,
+           supports_gradgrad=True,
+           supports_out=False,
+           autodiff_nonfusible_nodes=["aten::leaky_relu"],
+            skips=(
+                SkipInfo('TestCommon', 'test_variant_consistency_jit', dtypes=[torch.float32]),),),
```

Comment:
what was the issue with this test?

Reply:
in test_variant_consistency_jit function we have: check_alias_annotation where torch::jit::getOperatorAliasMap() get queried for leaky_relu, but leaky_relu is not part of that map:
https://github.com/pytorch/pytorch/blob/9def776cd6d775779db9839d942a01b4b082175c/torch/csrc/jit/passes/normalize_ops.cpp#L63